catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -125,6 +132,7 @@ const SearchForm = ({
     const sortOrder = inputValue(form['sortOrder']);
     const sort =
       sortOrder === 'asc' || sortOrder === 'desc' ? 'production.dates' : null;
+    const imageSearch = inputValue(form['searchType']) === 'imageSearch';
```

Comment:
I wonder if `searchType` is actually nicer?
You could just have a link that appends (`search=images`) onto the query.

Reply:
Think I agree but iirc this becomes a pain to exclude it from the URLs - probably worth returning to when it's out of the toggle